controllers/external-accounts.js

```diff
@@ -41,4 +44,50 @@ const getExternalAccountData = async (req, res) => {
   }
 };
 
-module.exports = { addExternalAccountData, getExternalAccountData };
+/**
+ * Gets all group-roles
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+const syncExternalAccountData = async (req, res) => {
+  try {
+    const discordUserData = await getDiscordMembers();
+    const rdsUserData = await getDiscordUsers();
```

Comment:
NIT: Make it 54,55 in one line if possible

Reply:
done